src/lib/learn-client/api/collection/index.ts

```diff
@@ -141,7 +143,7 @@ export async function getCollectionsBySection(section: string) {
 			console.warn(`No collections found for section query: ${section}`)
 		}
 
-		return res.result
+		return res.result.map(formatCollection)
```

Comment:
Needed to format this return data to make it consistent with other return shapes in the client.